src/com/projectkorra/projectkorra/ability/util/MultiAbilityManager.java

```diff
@@ -134,10 +134,7 @@ public static boolean hasMultiAbilityBound(final Player player, final String mul
 			return false;
 		}
 
-		if (playerAbilities.containsKey(player)) {
-			if (!playerBoundAbility.get(player).equals(multiAbility) && bPlayer.getBoundAbility() != null) {
-				return false;
-			}
+		if (playerAbilities.containsKey(player) && playerBoundAbility.get(player).equals(multiAbility)) {
 			return true;
 		}
 		return false;
```

Comment:
Could probably get away with just saying
```java
return playerAbilities.containsKey(player) && playerBoundAbility.get(player).equals(multiAbility)
```

Reply:
You're right

Reply:
Resolved